src/migrations/20220122121713-add-default-currency-walletinvoices.ts

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  async up(db) {
+    console.log("Begin up migration")
+    const result = await db
+      .collection("invoiceusers")
```

Comment:
Should we use this as an opportunity to rename the collection inline with the domain types to `walletinvoices`?